packages/hash/external-services/kratos/identity.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$id": "https://schemas.ory.sh/presets/kratos/quickstart/email-password/identity.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Person",
+  "type": "object",
+  "properties": {
+    "traits": {
+      "type": "object",
+      "properties": {
+        "emails": {
+          "type": "array",
```

Comment:
I've made this an array as we want to support multiple email addresses down the line, which is something that [appears to be supported by kratos](https://www.ory.sh/docs/kratos/concepts/credentials/username-email-password#use-case-multiple-emails-and-password).